.changeset/lucky-dingos-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-tools/validate': patch
+---
+
+Ignores dependencies imported from a `/scripts` directory, or `*.stories.js` file.
```

Comment:
don't we need stories for dev deps?

Reply:
`*.stories.js` is the generated story file. It may include some other dependencies if not properly built in rollup. This prevents false positives